metadata/ComputerVision__9__metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "changeLog": "Release v2022.10.10",
+  "cpu": 0,
+  "description": "Backend server for the Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
+  "displayName": "ComputerVision",
+  "gpu": 1,
+  "inputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation.",
+  "inputType": "JSON",
+  "memory": 0,
+  "mlPackageLanguage": "PYTHON36_CV",
+  "name": "ComputerVision",
+  "outputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation.",
+  "processorType": "GPU",
+  "projectId": "[project-id]",
+  "retrainable": false,
+  "stagingUri": "[staging-uri]",
+  "projectName": "UiPath Computer Vision",
+  "projectDescription": "Backend server for the Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
+  "tenantName": "UiPath",
+  "minAIFabricVersion": "22.10",
+  "languageVersion": 3,
+  "version": 4,
+  "customVersion": "22.10.1",
```

Comment:
Why is this `22.10.1` but above there is `Release v2022.10.10`? Can these versions match without any negative side-effects?

Reply:
I think you're right. I've changed to 22.10.10, since the same behaviour was also in 23.4/23.10.
@irinel-birsan-uipath can you please confirm that this is the correct way to do?

Reply:
we set it like `minAIFabricVersion":  "22.10",`
Anyway, this is onlt informative, there is no check done on AIC side :(